server/storage/mvcc/kvstore.go

```diff
@@ -238,7 +247,11 @@ func (s *store) compact(trace *traceutil.Trace, rev, prevCompactRev int64) (<-ch
 			s.compactBarrier(context.TODO(), ch)
 			return
 		}
-		s.hashes.Store(hash)
+		// Only store the hash value is the previous hash is completed, i.e. this compaction
```

Comment:
nit: looks like a small typo?

```suggestion
		// Only store the hash value if the previous hash is completed, i.e. this compaction
```